### PR TITLE
Fix SwiftLint errors

### PR DESCRIPTION
**Pull request by mokagio**
Builds on top of #12017 and addresses all the errors and some of the warnings SwiftLint picked up.

Recommend reviewing commit by commit.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">View files have been modified, but no screenshot or video is included in the pull request. Consider adding some for clarity.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 500 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Experiments/Experiments/FeatureFlag.swift [hunk 1]
```diff
@@ -149,7 +149,7 @@ public enum FeatureFlag: Int {
     case shareProductAI
```
Comment (iangmaia) [inline Experiments/Experiments/FeatureFlag.swift:149]: These errors are still being reported on CI:

- `WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift:1365:52`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayCompleteViewController.swift:135:32`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayInformationViewController.swift:199:28`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `WooCommerce/Classes/ViewRelated/Search/SearchViewController.swift:341:21`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `Yosemite/Yosemite/Model/Mocks/Graphs/ScreenshotsObjectGraph.swift:13:26`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `Yosemite/Yosemite/Model/Mocks/Graphs/ScreenshotsObjectGraph.swift:14:31`: Localized strings should include a description giving context for how the string is used. (localization_comment)
- `Yosemite/Yosemite/Model/Mocks/Graphs/ScreenshotsObjectGraph.swift:15:31`: Localized strings should include a description giving context for how the string is used. (localization_comment)

**Review by iangmaia — APPROVED** [review]

File: Experiments/Experiments/FeatureFlag.swift (hunk 1 above)
Comment (mokagio) [reply Experiments/Experiments/FeatureFlag.swift:149]: 🤔 interesting to see that CI considers them error when they are but warnings on my end.

Comment (mokagio) [reply Experiments/Experiments/FeatureFlag.swift:149]: Actually, not interesting at all, given CI uses `--strict` 😅

**Pull request merged**